Cesium.CodeGen/Ir/Declarations/LocalDeclarationInfo.cs

```diff
@@ -208,20 +208,46 @@ public static LocalDeclarationInfo Of(IReadOnlyList<IDeclarationSpecifier> speci
+        return new PrimitiveType(
+            (p1, p2, p3, p4) switch
             {
-                "char" => PrimitiveTypeKind.Char,
-                "int" => PrimitiveTypeKind.Int,
-                "void" => PrimitiveTypeKind.Void,
-                var unknown =>
-                    throw new NotImplementedException($"Not supported yet type specifier: {unknown}.")
-            }),
-            { Count: 2 } when typeNames[0] == "unsigned" && typeNames[1] == "char" =>
-                new PrimitiveType(PrimitiveTypeKind.UnsignedChar),
-            _ => throw new NotImplementedException(
-                $"Simple type specifiers are not supported: {string.Join(" ", typeNames)}")
-        };
+                ("signed", "long", "long", "int") => PrimitiveTypeKind.SignedLongLongInt,
+                ("unsigned", "long", "long", "int") => PrimitiveTypeKind.UnsignedLongLongInt,
+                ("signed", "short", "int", _) => PrimitiveTypeKind.SignedShortInt,
+                ("signed", "long", "int", _) => PrimitiveTypeKind.SignedLongInt,
+                ("signed", "long", "long", _) => PrimitiveTypeKind.SignedLongLong,
+                ("long", "long", "int", _) => PrimitiveTypeKind.LongLongInt,
+                ("unsigned", "short", "int", _) => PrimitiveTypeKind.UnsignedShortInt,
+                ("unsigned", "long", "int", _) => PrimitiveTypeKind.UnsignedLongInt,
+                ("unsigned", "long", "long", _) => PrimitiveTypeKind.UnsignedLongLong,
+                ("signed", "char", _, _) => PrimitiveTypeKind.SignedChar,
+                ("signed", "short", _, _) => PrimitiveTypeKind.SignedShort,
+                ("short", "int", _, _) => PrimitiveTypeKind.ShortInt,
+                ("signed", "int", _, _) => PrimitiveTypeKind.SignedInt,
+                ("signed", "long", _, _) => PrimitiveTypeKind.SignedLong,
+                ("long", "int", _, _) => PrimitiveTypeKind.LongInt,
+                ("long", "long", _, _) => PrimitiveTypeKind.LongLong,
+                ("long", "double", _, _) => PrimitiveTypeKind.LongDouble,
+                ("unsigned", "char", _, _) => PrimitiveTypeKind.UnsignedChar,
+                ("unsigned", "short", _, _) => PrimitiveTypeKind.UnsignedShort,
+                ("unsigned", "int", _, _) => PrimitiveTypeKind.UnsignedInt,
+                ("unsigned", "long", _, _) => PrimitiveTypeKind.UnsignedLong,
+                ("void", _, _, _) => PrimitiveTypeKind.Void,
+                ("char", _, _, _) => PrimitiveTypeKind.Char,
+                ("short", _, _, _) => PrimitiveTypeKind.Short,
+                ("signed", _, _, _) => PrimitiveTypeKind.Signed,
+                ("int", _, _, _) => PrimitiveTypeKind.Int,
+                ("unsigned", _, _, _) => PrimitiveTypeKind.Unsigned,
+                ("long", _, _, _) => PrimitiveTypeKind.Long,
+                ("float", _, _, _) => PrimitiveTypeKind.Float,
+                ("double", _, _, _) => PrimitiveTypeKind.Double,
```

Comment:
Could you please replace `_` with `null` here?

Otherwise, this may match invalid code patterns like `long int int` and such.